### PR TITLE
Added support and test for LDRWroX

### PR DESCRIPTION
**Pull request by tanmaytirpankar**
clang-formatted

### CODE REVIEW

File: backend_tv/lifter.cpp [hunk 1]
```diff
@@ -2842,7 +2872,25 @@ class arm2llvm {
       updateReg(added, ptrReg);
       break;
     }
+    case AArch64::LDRWroX: {
+      unsigned size;
+      if (opcode == AArch64::LDRWroX)
+        size = 4;
+      else
+        assert(false);
+      auto &op0 = CurInst->getOperand(0);
```
Comment (regehr) [inline backend_tv/lifter.cpp:2881]: this whole first part isn't needed -- you get the operands and do the assertions over again in the helper function

Comment (regehr) [reply backend_tv/lifter.cpp:2881]: sorry I mean lines 2881-2887 should be removed

**Comment by tanmaytirpankar** [comment]
Fixed

**Comment by regehr** [comment]
awesome!!

**Pull request merged**